models/app.models.js

```diff
@@ -26,3 +26,17 @@ exports.getArticlesFromDatabase = (article_id) => {
     else return rows;
   });
 };
+
+exports.getArticlesFromDatabase = () => {
+  let query = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count
+  FROM articles
+  LEFT JOIN comments ON comments.article_id = articles.article_id
+  GROUP BY articles.article_id
+  ORDER BY articles.created_at DESC`;
+  return db.query(query).then(({ rows }) => {
+    return rows.map((article) => {
+      article.comment_count = Number(article.comment_count);
+      return article;
```

Comment:
This is an interesting way to do this, can you think of a way to do this using the query itself instead JavaScript?